features/step_definitions/rest_register_steps.rb

```diff
@@ -0,0 +1,13 @@
+Then(/^I should see "([^"]*)" link$/) do |link|
+    expect(page).to have_link link
+end
+
+Then(/^I click on the "([^"]*)" link$/) do |link|
+  click_link(link)
+end
+
+Given(/^I am logged in with "([^"]*)", password '(\d+)' and password_confirmation '(\d+)'$/) do |email, pwd, pwd_confirm|
+  current_user = FactoryGirl.create(:user, email: email, password: pwd, password_confirmation: pwd_confirm )
```

Comment:
Even in this case I think that using variable name `current_user` feels weird. I would have chosen `user`. Not a blocket though. ;-)